diff -up mutt-1.10.1/sendlib.c.mutt-1.9.3-1_coverity_187_188_189_190 mutt-1.10.1/sendlib.c
--- mutt-1.10.1/sendlib.c.mutt-1.9.3-1_coverity_187_188_189_190	2018-11-26 12:34:51.007894823 +0100
+++ mutt-1.10.1/sendlib.c	2018-11-26 12:35:08.374143006 +0100
@@ -1815,11 +1815,15 @@ static int write_one_header (FILE *fp, i
 	      NONULL(pfx), valbuf, max, wraplen));
     if (pfx && *pfx)
       if (fputs (pfx, fp) == EOF)
+      {
+        FREE(&valbuf);
 	return -1;
+      }
     if (!(t = strchr (valbuf, ':')))
     {
       dprint (1, (debugfile, "mwoh: warning: header not in "
 		  "'key: value' format!\n"));
+      FREE(&valbuf);
       return 0;
     }
     if (print_val (fp, pfx, valbuf, flags, mutt_strlen (pfx)) < 0)
@@ -1861,7 +1865,11 @@ static int write_one_header (FILE *fp, i
 	      "max width = %d > %d\n",
 	      NONULL(pfx), valbuf, max, wraplen));
     if (fold_one_header (fp, tagbuf, valbuf, pfx, wraplen, flags) < 0)
+    {
+      FREE (&tagbuf);
+      FREE (&valbuf);
       return -1;
+    }
     FREE (&tagbuf);
     FREE (&valbuf);
   }
