diff -up mutt-1.10.1/send.c.mutt-1.9.3-1_coverity_181 mutt-1.10.1/send.c
--- mutt-1.10.1/send.c.mutt-1.9.3-1_coverity_181	2018-11-26 12:08:42.615216677 +0100
+++ mutt-1.10.1/send.c	2018-11-26 12:08:52.007351542 +0100
@@ -243,7 +243,7 @@ static int edit_envelope (ENVELOPE *en)
       if (ascii_strncasecmp ("subject:", uh->data, 8) == 0)
       {
 	p = skip_email_wsp(uh->data + 8);
-	strncpy (buf, p, sizeof (buf));
+	strfcpy (buf, p, sizeof (buf));
       }
     }
   }
